front-end/src/telemetry/notification.jsx

```diff
@@ -27,7 +27,8 @@ export default class NotificationSettings extends React.Component {
   updateTo () {
     return function (ev) {
       const config = this.state.config
-      config.to = ev.taregt.value.split(',').map(s => s.trim())
+      let recipients = ev.target.value.split()
```

Comment:
The only actual change I see is removing the argument to split.  This should just return the array with a single element, although splitting on ',' should do pretty much the same.